### PR TITLE
Fix hubspot request logging. again

### PR DESCRIPTION
**Pull request by TylerSheffels**
Will almost certainly fix the copious number of error emails.

@czue, @dannyroberts

### CODE REVIEW

File: corehq/apps/analytics/tasks.py [hunk 1]
```diff
@@ -369,3 +365,14 @@ def _track_periodic_data_on_kiss(submit_json):
...
+
+def _log_response(data, response):
+    try:
+        response_text = json.dumps(response.json(), indent=2, sort_keys=True)
+    except:
+        response_text = response.status_code
+    logger.debug('Sent this data to HS: %s \nreceived: %s' %
+                 (json.dumps(data, indent=2, sort_keys=True),
+                  response_text))
+
```
Comment (dimagimon) [inline corehq/apps/analytics/tasks.py]: W391 blank line at end of file

**Comment by snopoke** [comment]
lol

**Comment by czue** [comment]
i like the confidence

File: corehq/apps/analytics/tasks.py [hunk 2]
```diff
@@ -369,3 +365,13 @@ def _track_periodic_data_on_kiss(submit_json):
         s3_connection.upload(filename, f, 'kiss-uploads')
 
     os.remove(filename)
+
+
+def _log_response(data, response):
+    try:
+        response_text = json.dumps(response.json(), indent=2, sort_keys=True)
+    except:
```
Comment (czue) [inline corehq/apps/analytics/tasks.py]: `except Exception` is probably preferred here since this will also swallow keyboard interrupts and such

Comment (TylerSheffels) [inline corehq/apps/analytics/tasks.py]: https://github.com/dimagi/commcare-hq/pull/9061

**Pull request merged**